spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -125,6 +126,17 @@ def argv(self):
             # If executable is a conda environment, use "run" subcommand to
             # activate it and run spyder-kernels.
             conda_exe = find_conda()
+            if not conda_exe:
+                # Raise error since we were unable to determine the path to
+                # the conda executable (e.g conda was installed in a
+                # non-standard location).
```

Comment:
```suggestion
                # the conda executable (e.g when Anaconda/Miniconda was
                # installed in a non-standard location).
```

Improve comment a bit.